src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -158,8 +158,114 @@ mocha.describe('readings API', () => {
 
 				// Add C10 here
 
-				// Add C11 here
-
+				mocha.it('C11: 1 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as BTU reverse conversion', async () => {
+					const unitData = [
```

Comment:
Please take a look at src/server/test/web/readingsBarMeterQuantity.js for B11 on how the units & conversions are created with concat. While this works that is preferred since it reuses the standard definitions.